skins/oasis/js/WikiaFooter.js

```diff
@@ -12,12 +15,12 @@ var WikiaFooterApp = {
 			this.footer = $( '#WikiaFooter' );
 		}
 		this.toolbar = this.footer.children( '.toolbar' );
-		this.gn = $( '.global-notification' );
+		this.bn = $( '.banner-notification' );
```

Comment:
Actually it looks like we can remove references to banner notifications from this file all together, since we don't need the scroll event or the resize event for BN.